include/nbl/builtin/glsl/ext/FFT/default_compute_fft.comp

```diff
@@ -0,0 +1,127 @@
+#ifndef _NBL_GLSL_EXT_DEFAULT_COMPUTE_FFT_INCLUDED_
+#define _NBL_GLSL_EXT_DEFAULT_COMPUTE_FFT_INCLUDED_
+
+// WorkGroup Size
+
+#ifndef USE_SSBO_FOR_INPUT
+#error "USE_SSBO_FOR_INPUT should be defined."
+#endif
+
+#ifndef _NBL_GLSL_EXT_FFT_BLOCK_SIZE_X_DEFINED_
+#define _NBL_GLSL_EXT_FFT_BLOCK_SIZE_X_DEFINED_ 256
+#endif
+
+#ifndef _NBL_GLSL_EXT_FFT_BLOCK_SIZE_Y_DEFINED_
+#define _NBL_GLSL_EXT_FFT_BLOCK_SIZE_Y_DEFINED_ 1
+#endif
+
+#ifndef _NBL_GLSL_EXT_FFT_BLOCK_SIZE_Z_DEFINED_
+#define _NBL_GLSL_EXT_FFT_BLOCK_SIZE_Z_DEFINED_ 1
+#endif
+
+#define _NBL_GLSL_WORKGROUP_SIZE_ (_NBL_GLSL_EXT_FFT_BLOCK_SIZE_X_DEFINED_*_NBL_GLSL_EXT_FFT_BLOCK_SIZE_Y_DEFINED_*_NBL_GLSL_EXT_FFT_BLOCK_SIZE_Z_DEFINED_)
+
+layout(local_size_x=_NBL_GLSL_EXT_FFT_BLOCK_SIZE_X_DEFINED_, local_size_y=_NBL_GLSL_EXT_FFT_BLOCK_SIZE_Y_DEFINED_, local_size_z=_NBL_GLSL_EXT_FFT_BLOCK_SIZE_Z_DEFINED_) in;
```

Comment:
you should always run a `WG_SIZEx1x1` workgroup to perform an FFT there's no point to non-1D shapes